openapi/components/schemas/Agency.yaml

```
@@ -8,6 +8,12 @@ properties:
     type: string
     description: The full name of the agency.
     example: Department of Transportation
+  order:
+    type: string
+    description: >-
+      The criteria used to sort the results using the agency initials in ascending order.
+    example: DCP
```

The order informs the field on which the list is sorted, rather than a specific entry in the field.

So, it says "initials" instead of "DCP"

```suggestion
    example: initials
```